### ISSUE TITLE
Seg Fault when you have a titlebar menu button in Plasma

### ISSUE THREAD

**Issue by m3talslug** [Heroic-Games-Launcher/HeroicGamesLauncher#2584]
### Describe the bug

In Plasma Desktop Environment.  If you have Window Decorations > Titlebar buttons > and use the "Title Bar button".  Enabling this in Plasma will cause the Heroic Launcher to seg fault when you click on any of the menu buttons i.e. File, Edit, etc

### Add logs

```text
Plasma Desktop Environment:

1) Settings > Appearance > Window Decorations > Titlebar buttons
2) Drag the "Application Menu" button to the title bar
3) Launch Heroic launcher
4) Click on the file menu
5)  -- seg fault
```

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

I expect it to work with the application menu button enabled

### Heroic Version

Latest Stable

### System Information

- SteamOS Latest Stable (Arch Linux)

**Comment by Nocccer** [comment]
Sounds like a electron error with plasma.

**Comment by m3talslug** [comment]
I don't think so.  This never happened in older builds of Heroic and no other app seg faults.  It's a bug with how Heroic handles the OS's window decoration in Linux.

**Comment by Nocccer** [comment]
Actually why there is a window decoration like file, edit in first place? Ahould it jot just be close, minimize or maximize. Probably we need a log of that segfault and also a image what you mean with window decoration

**Comment by flavioislima** [comment]
I will try replicate this during the week. But if that's the case we need to report this bug to the electron team. I don't think we can fix it on our end.
Might also be fixed already since we have updated our electron version on the main branch.

**Comment by m3talslug** [comment]
![Screenshot_20230403_232011](https://user-images.githubusercontent.com/48932579/229678927-af6d9f86-7a85-4661-b949-16f99f8cfb23.png)

Enable the decoration in the screenshot. It's a stock plasma decoration. Run the Heroic program and then try clicking on the menu items.
